ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -34,6 +34,9 @@ chain:
     - name: HYPERSHIFT_INFRA_AVAILABILITY_POLICY
       default: "SingleReplica"
       documentation: "Availability policy for infrastructure services in guest cluster. Supported options: SingleReplica, HighlyAvailable, default SingleReplica"
+    - name: HYPERSHIFT_INSTANCE_TYPE
```

Comment:
nit: May want to add `_x86` or `_AMD64` or something like that to the name. Sometime we may have multi-arch hosted clusters/NodePools. There is a PR to implement that is on hold. The instance type will matter if we are using ARM over x86.